cmd/utils/cmd.go

```diff
@@ -214,6 +216,20 @@ func ExportAppendChain(blockchain *blockchain.BlockChain, fn string, first uint6
 	return nil
 }
 
+func NtpCheckWithLocal() error {
+	local := time.Now().UTC().Unix()
+	ntp, err := ntpclient.GetNetworkTime("time.nist.gov", 123)
```

Comment:
Can we make this domain as a configuration parameter?

Reply:
@kjhman21 I will add the flags to enable checking ntp and passing the ntp server url in the PR following the #1638